aussie_clock.py

```diff
@@ -114,6 +115,25 @@ async def goob_schedule_upd():
                         await channel.send(schedule_message)
 
 
+async def birthday_check():
+    birthdays = json.loads(open("birthdays.json", "r").read())
+    for b_day in birthdays:
+        user = client.get_user(b_day)
+        today = datetime.today()
+        b_day = datetime.fromtimestamp(birthdays[b_day])
+        if today.month == b_day.month:
+            if today.day == b_day.day:
+                if today.hour == 14 and today.minute == 0:
+                    for guild in client.guilds:
+                        for channel in guild.text_channels:
+                            if channel.topic is not None:
+                                if "YouTube Ping" in channel.topic:
+                                    if b_day.year > 1:
+                                        await channel.send(f'Merry {today.year - b_day.year}th Birthmas {user.mention}!')
+                                    else:
+                                        await channel.send(f'Merry Birthmas {user.mention}!')
```

Comment:
Overall, this code is far too indented for my liking. It could be worthwhile setting up some CI (using something like `flake8` to lint the code) to automatically detect these kinds of issues